demisto_sdk/commands/test_content/test_modeling_rule/tests/test_modeling_rule_test.py

```diff
@@ -926,14 +926,20 @@ def test_the_test_modeling_rule_command_results_with_ignored_validations(self, p
         pack.create_modeling_rule(DEFAULT_MODELING_RULE_NAME, rules=ONE_MODEL_RULE_TEXT)
         mrule_dir = Path(pack._modeling_rules_path / DEFAULT_MODELING_RULE_NAME)
         path_to_fake_test_data_file = (
-                Path(__file__).parent / "test_data/fake_test_data_file.json"
+            Path(__file__).parent / "test_data/fake_test_data_file.json"
         )
 
         test_data_file = pack.modeling_rules[0].testdata
 
         fake_test_data = TestData.parse_file(path_to_fake_test_data_file.as_posix())
         test_data_file.write_as_text(fake_test_data.json(indent=4))
-        test_data_file.update({"ignored_validations": [Validations.SCHEMA_TYPES_ALIGNED_WITH_TEST_DATA.value]})
+        test_data_file.update(
+            {
+                "ignored_validations": [
+                    Validations.SCHEMA_TYPES_ALIGNED_WITH_TEST_DATA.value
```

Comment:
If your class inherits from str as well, then you will able to use it without `.value`

class Validations(Enum, str):
    SCHEMA_TYPES_ALIGNED_WITH_TEST_DATA = "schema_test_data_types"

```suggestion
                    Validations.SCHEMA_TYPES_ALIGNED_WITH_TEST_DATA
```